hackernews/program.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
Currently, in the kittehub repo, we have a docstring at the top of every python file that describes at a high level how the workflow works. Is this redundant? Do we want to keep kittehub consistent?

@daabr @haimzlato